package.json

```
@@ -8,8 +8,8 @@
   "scripts": {
     "build": "rimraf ./dist && tsc && git rev-parse HEAD > BUILD_SHA",
     "build:test": "tsc --project tsconfig.test.json",
-    "lint": "eslint \"*/**/*.{js,ts,tsx}\"",
-    "lint:fix": "prettier --loglevel warn --write \"src/**/*.{ts,tsx}\" && eslint \"*/**/*.{js,ts,tsx}\" --quiet --fix",
+    "lint": "npx eslint",
+    "lint:fix": "npx eslint --fix",
```

```suggestion
    "lint": "eslint",
    "lint:fix": "eslint --fix",
```

Don't need `npx` do we if eslint is already a dependency?

---

Let me test it.